internal/providers/terraform/cloud.go

```diff
@@ -38,135 +29,10 @@ func cloudAPI(host string, path string, token string) ([]byte, error) {
 	defer resp.Body.Close()
 
 	if resp.StatusCode == 401 {
-		return []byte{}, ErrInvalidCloudToken
+		return []byte{}, modules.ErrInvalidCloudToken
```

Comment:
Seems odd we are referencing the `hcl/modules` package from the `cloud` and `dir_provider`. I assume the errors and functions were moved there to avoid circular dependencies? Not sure if it's worth introducing a separate package for these.